modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4447,6 +4454,88 @@ public void testGetObjectEntriesWithAggregationFacets() throws Exception {
 			1, String.valueOf(parentObjectEntry2.getId()), page);
 	}
 
+	@Test
+	public void testGetObjectEntriesWithScopeDepot() throws Exception {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
+
+		DepotEntry depotEntry = _depotEntryLocalService.addDepotEntry(
+			HashMapBuilder.put(
+				LocaleUtil.getDefault(), RandomTestUtil.randomString()
+			).build(),
+			HashMapBuilder.put(
+				LocaleUtil.getDefault(), RandomTestUtil.randomString()
+			).build(),
+			serviceContext);
+
+		Group group = _groupLocalService.getGroup(
+			companyId, GroupConstants.GUEST);
+
+		_depotEntryGroupRelLocalService.addDepotEntryGroupRel(
+			depotEntry.getDepotEntryId(), group.getGroupId());
+
+		_defaultObjectEntryManager.addObjectEntry(
+			_simpleDTOConverterContext, _objectDefinition1,
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"textObjectFieldName", "Able1"
+					).put(
+						"textObjectFieldNameExtension", "Baker1"
+					).build();
+				}
+			},
+			ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		_objectDefinition2 = _createObjectDefinition(
+			Arrays.asList(
+				new TextObjectFieldBuilder(
+				).indexed(
+					true
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).name(
+					"textObjectFieldName"
+				).build()),
+			ObjectDefinitionConstants.SCOPE_DEPOT);
+
+		_objectDefinitionSettingLocalService.addObjectDefinitionSetting(
+			TestPropsValues.getUserId(),
+			_objectDefinition2.getObjectDefinitionId(),
+			ObjectDefinitionSettingConstants.NAME_ACCEPTED_GROUP_IDS,
+			String.valueOf(depotEntry.getGroupId()));
+
+		com.liferay.object.model.ObjectEntry objectEntry =
+			_objectEntryLocalService.addObjectEntry(
+				TestPropsValues.getUserId(), depotEntry.getGroupId(),
+				_objectDefinition2.getObjectDefinitionId(),
+				ObjectEntryFolderConstants.
+					PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT,
+				null,
+				HashMapBuilder.<String, Serializable>put(
+					"textObjectFieldName", "Able2"
+				).put(
+					"textObjectFieldNameExtension", "Baker2"
+				).build(),
+				serviceContext);
+
+		Page<ObjectEntry> page = _defaultObjectEntryManager.getObjectEntries(
+			companyId, _objectDefinition2, group.getGroupKey(), null,
+			dtoConverterContext, StringPool.BLANK, null, StringPool.BLANK,
+			new Sort[] {SortFactoryUtil.create("createDate", false)});
+
+		List<ObjectEntry> objectEntries = (List<ObjectEntry>)page.getItems();
+
+		Assert.assertEquals(objectEntries.toString(), objectEntries.size(), 1);
+
+		Assert.assertEquals(
+			_defaultObjectEntryManager.getObjectEntry(
```

Comment:
you won't need getting the entry again after using` _defaultObjectEntryManager.addObjectEntry ` in my previous comment